mobie/image_data.py

```diff
@@ -74,20 +75,30 @@ def add_bdv_image(xml_path, root, dataset_name,
 
     # get the setup ids and check that image_name is compatible
     setup_ids = bdv_metadata.get_setup_ids(xml_path)
+
     if image_name is None:
         image_name = [None] * len(setup_ids)
     else:
         if isinstance(image_name, str):
             image_name = [image_name]
+
     assert len(image_name) == len(setup_ids)
 
     data_path = bdv_metadata.get_data_path(xml_path, return_absolute_path=True)
     for setup_id, name in zip(setup_ids, image_name):
+        move_only = False
 
         # get the key for the input data format
         input_format = bdv_metadata.get_bdv_format(xml_path)
```

Comment:
This should be done outside of the loop, since it will be the same for each setup_id.
(Looks like this was wrong in my code already.)

Reply:
I put it inside the loop for the very crazy idea of having one XML pointing to different formats. But that cannot happen (I confused it with what we do with views).
In fact, moving only makes sense if there is only one setup ID. I remember that MoBIE had the limitation of only being able to load one Setup from an XML at once. Or does that work now? If not, we might need another check here.